tests/src/runtimeApi/stream/hipStreamAddCallback.cpp

```diff
@@ -23,41 +23,69 @@ THE SOFTWARE.
  * HIT_END
  */
 
-// Test under-development. Call hipStreamAddCallback function and see if it works as expected.
-
+#include <stdio.h>
+#include <unistd.h>
```

Comment:
unistd.h is a POSIX header. With supporting Windows in the future it should be replaced with one or multiple C/C++ headers and also supplemented with platform ifdefs, for instance:

```c++
+#ifdef _WIN32
+#include <windows.h>
+#define sleep(x) Sleep(x)
+#else
+#include <unistd.h>
+#endif 
```

Reply:
It looks like unistd.h is added for sleep(1), so why don't replace it with STL's ones: 

```c++
std::this_thread::sleep_for(std::chrono::milliseconds(1000));
```

Reply:
@emankov: Updated the code to use sleep_for instead of unistd sleep.